code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/magic/wand
 	name = "wand of nothing"
-	desc = "It's not just a stick, it's a MAGIC stick!"
+	desc = "It's not just a stick, it's a MAGIC stick! You shouldn't have this."

```

Comment:
This is also the wand of nothing's description sooo do something about that.

Reply:
fixed